### ISSUE TITLE
Fuseau horaire sur les modifications

### ISSUE THREAD

**Issue by AntoineAugusti** [etalab/data.gouv.fr#171]
## Description du bug
Les modifications sont parfois affichées dans le futur quand on consulte l'interface avec un fuseau horaire différent de Europe/Paris

## Comment reproduire le bug
Étapes à suivre pour faire apparaître le bug :
1. Etre sur un fuseau horaire différent de Europe/Paris
2. Modifier un JDD
3. Consulter les dernières activités sur une page d'organisation
4. Constater le bug

![image](https://user-images.githubusercontent.com/295709/81570481-a577d100-936e-11ea-9130-94c5223a2072.png)

## Comportement attendu
L'heure affichée est l'heure locale de l'utilisateur.

**Comment by abulte** [comment]
Yup :-/ Also here, don't know why it's closed https://github.com/opendatateam/udata/issues/1046

**Comment by fchabouis** [comment]
Just a reminder for us and you : I see in transport.data.gouv code that we have adapted to this specificity of your api and treat dates coming from data.gouv as naive datetimes. I think that of course it would be best to have dates with time zones, but the day you implement it, we'll need to adapt our code accordingly.
